packages/skeleton-svelte/src/lib/components/ProgressRing/types.ts

```diff
@@ -0,0 +1,72 @@
+import type { Snippet } from 'svelte';
+
+export interface ProgressRingProps {
+	/** Set the value */
+	value?: number;
+	/** Set the maximum value */
+	max?: number;
+	/** Set the stroke size (px) */
+	strokeWidth?: number;
+	/** Defines the shape of the meter */
+	strokeLinecap?: 'inherit' | 'butt' | 'round' | 'square';
+	/** Set the aria-labelledby value */
+	labelledby?: string;
+
+	// Root (Figure) ---
+	/** Set the root base classes */
+	base?: string;
+	/** Set the root size classes */
+	size?: string;
+	/** Provide arbitrary classes to the root element */
+	classes?: string;
+
+	// Slot ---
```

Comment:
Yeah it's a tough one, because Svelte used to call these slots, now they're snippets, and React calls them Children. So there's no de-facto option. I might switch to Children here.

Reply:
``childBase`` and ``childClasses`` doesn't feel good either though.
``Areas`` maybe since I there's cases where different areas can take different snippets?

Reply:
So between React and Svelte v5, the term `children` seems to be the standard:

- React: https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children
- Svelte 5: https://svelte-5-preview.vercel.app/docs/snippets#typing-snippets

Note that in Svelte `children` is merely a **convention** for the default Snippet. You can actually call it whatever you want. But we're already following this pattern, so I say we lean into it.

I'm going to opt for `childrenBase` and `childrenClasses` respectively unless anyone can provide a strong argument otherwise.